src/main/webapp/app/shared/search/search.component.scss

```diff
@@ -0,0 +1,29 @@
+.search-block {
+  display: flex;
+  padding-top: 1.25rem;
+
+  .search-field {
+    margin-left: auto;
+    width: 200px;
+  }
+
+  .mat-icon {
+    color: grey;
+  }
+
+  ::ng-deep {
```

Comment:
this is oudated, what would you do instead?

Reply:
I replaced the **ng-deep** by turning the view encapsulation off but it is also not a best practice either. 
IMHO it would be better that we move the logic to global SCSS file to create some SCSS mixins. WDYT?